src/auth/README.md

```diff
@@ -1,10 +1,21 @@
-# Blockstack Auth
+# Blockstack Authentication
 
-SSO and authentication without 3rd parties or remote servers
+Blockstack Authentication provides single sign on and authentication without third parties or remote servers.
 
+* [Overview](#overview)
 * [Quickstart](#quickstart)
-* [Operations](#operations)
-* [Examples](#examples)
+* [API Reference](http://blockstack.github.io/blockstack.js/index.html#authentication)
+* [User flow](#user-flow)
+* [Manifest file](#manifest-file)
+* [Key pairs](#key-pairs)
+* [Scopes](#scopes)
+* [Authentication tokens](#authentication-tokens)
+* [`blockstack:` custom protocol handler](#blockstack-custom-protocol-handler)
+* [Adding Blockstack Authentication to your app](#adding-blockstack-authentication-to-your-app)
+
+## Overview
+
+Blockstack Authentication is a bearer token-based authentication system. From an app user's perspective, it functions similar to legacy third-party authentication techniques that they're familiar with. From an app developers' perspective it's a bit different than a typical client-server based OAuth flow that you've seen with centralized sign in services in that authentication happens entirely on the client side.
```

Comment:
"From an app developers' perspective it's a bit different than a typical client-server based OAuth flow that you've seen with centralized sign in services in that authentication happens entirely on the client side." 

That sentence has a lot of clauses to parse. Maybe reword like:

"For an app developer, the flow is a bit different from the typical client-server flow of centralized sign in services (e.g., OAuth). Rather, with Blockstack, the authentication flow happens entirely client-side."

Reply:
Like this.